### DISCUSSION TITLE
Installing Birt on Weblogic 14

### DISCUSSION THREAD

**Discussion by EAwesomeK** [eclipse-birt/birt#1214] [Q&A]
I attempt to install the webviewer on Weblogic using the instructions that I found for POJO and it deploys but I can't get any reports to show up.  I get:

Error 404--Not Found

I am using localhost:7001/Birt which is the deployed .   Is it actually supported?

**Comment by wimjongman** [comment]
404 means that you point it to a valid server, but there is no content on that path. It should be something like:

* http://localhost:7001/birt/index.jsp
* http://localhost:7001/birt/run?__report=Report\SalesInvoice.rptdesign 

#### Docs
* https://eclipse.github.io/birt-website/docs/integrating/viewer-setup/
* https://wiki.eclipse.org/BirtPOJO_Viewer_WebLogic_Deployment

**Reply by EAwesomeK** [reply]
Yes I know and should have been more clear.  Followed the instructions in both those docs previously and I have the same result for the URL' as well HOWEVER the weblogic console says that it is started and the URL should be: 

http://192.168.0.90:7001/Birt                 |                Default url on server AdminServer

What version of Weblogic is supported?  We have tried older versions as well with other problems.    In this case we are using 14.1.1.   The JDK is supposed to be 11 or greater correct?

**Reply by EAwesomeK** [reply]
Does anyone have an answer for this?  Has it actually been used with Weblogic?

**Reply by hvbtup** [reply]
Is anybody actually using WebViewer? I don't, but to me it seems that the instructions in the aforementioned docs are outdated.
I mean, the POJO runtime already contains a file `birt.war`, so why isn't it used?

**Reply by claesrosell** [reply]
There were issues reported regarding the WebViewer around the 4.9 release, some of them was fixed as well. That was with Tomcat though.
@EAwesomeK are there any warnings or errors in any logs?

**Reply by EAwesomeK** [reply]
no errors or warning in the logs.  HOWEVER, it does run fine on Linux, it does NOT run on Weblogic in Windows.  I am using the Birt.War in all instances.  It sounds like a problem with windows.  How is it being tested?

**Reply by hvbtup** [reply]
It cannot be tested automatically, because WebLogic is not Open Source. I guess it is not tested with WebLogic. From the error message you posted on Stackoverflow (why not here?) it seems that the colon after the "drive letter" `C:` is causing issues. Can it be that you mix up virtual paths with file system paths somehow?

**Reply by EAwesomeK** [reply]
Hi - Thanks for the reply.  All I did was just deploy as the instructions in POJO recommend.  IF there is anywhere in the setup that I can go and modify the paths it would be appreciated if you let me know.  As I said it runs fine in Linux with no changes.
